docs/src/turing.md

```diff
@@ -1,101 +1,105 @@
-## Parameter Estimation with Turing
+# Parameter Estimation with Turing
 
-It is possible to use [Turing.jl](https://turinglang.org/stable/) to perform Bayesian parameter estimation on models defined in SequentialSamplingModels.jl. Below, we show you how to estimate the parameters for the Linear Ballistic Accumulator (LBA).
 
-```@setup Turing 
```

Comment:
@DominiqueMakowski, the `@setup Turing`  code is necessary to run the model and generate the output for each block. We don't have to do this, but I like to run the code to ensure that the examples work.  You can test the documentation locally using the following steps:

1. In the REPL, type `;` to enter shell mode and cd to the root folder of sequentialsamplingmodels.
2. Escape out of shell mode and type `]` to enter package mode. 
3. type  `activate ./docs`  to activate the documentation environment
4. Install LiveServer to your global environment and type `using LiveServer; servedocs()`
5. click on the url at the end of the output to view in your browser. 

You will see that the example does not run because the code in the `@setup` macro was deleted. 

Can you make the following modifications? Can you restore the original example so we can have a simple example, and add your advanced example to a seperate file? Lastly, in `docs/make.jl` can you add the Turing examples to a nested menu similar to the models? Thanks!